src/applications/static-pages/facilities/FacilityAppointmentWaitTimesWidget.jsx

```diff
@@ -17,9 +17,8 @@ export class FacilityAppointmentWaitTimesWidget extends React.Component {
     // find service in waitTimes object as a key
     const time = waitTimes[serviceKey];
     // return waitTimes for service
-    return time
-      ? `${time[established ? 'established' : 'new'].toFixed(0)} days`
-      : '';
+    const waitTime = time[established ? 'established' : 'new'];
+    return time && waitTime ? `${waitTime.toFixed(0)} days` : '';
```

Comment:
```suggestion
    return waitTime ? `${waitTime.toFixed(0)} days` : '';
```
if `time` is falsey, it'll throw an error when trying to reference the property on it in line 20.

Reply:
very true. thanks :smile: